src/Appwrite/Extend/Exception.php

```diff
@@ -116,9 +116,16 @@ class Exception extends \Exception
     public const STORAGE_INVALID_CONTENT_RANGE     = 'storage_invalid_content_range';
     public const STORAGE_INVALID_RANGE             = 'storage_invalid_range';
 
+    /** VCS */
+    public const INSTALLATION_NOT_FOUND              = 'installation_not_found';
+    public const PROVIDER_REPOSITORY_NOT_FOUND       = 'provider_repository_not_found';
+    public const REPOSITORY_NOT_FOUND                = 'repository_not_found';
+    public const PROVIDER_CONTRIBUTION_CONFLICT      = 'provider_contribution_conflict';
```

Comment:
We need to prefix error codes of a service with the service name. so `VCS_INSTALLATION_NOT_FOUND`

Reply:
We dont do that for document. It's not DATABASE_DOCUMENT_NOT_FOUND.